File: java/MDC.java
import java.util.Scanner;

public class MDC{
    void calcularMDC(){
        Scanner dado = new Scanner(System.in);

        System.out.println("Cálculo do MDC.");

        System.out.println("Digite o primeiro número: ");
        int a = dado.nextInt();
        System.out.println("Digite o segundo número: ");
        int b = dado.nextInt();

        int x = a, y = b;

        while (y != 0) {
            int resto = x % y;
            x = y;
            y = resto;
        }

        System.out.println("O MDC entre " + a + " e " + b + " é " + x + ".\n");
    }
}

File: JavaScript/MDC.js
console.log("Cálculo do MDC.");
let a = parseInt(prompt("Digite o primeiro número: "));
let b = parseInt(prompt("Digite o segundo número: "));

let x = a, y = b;

while (y !== 0) {
    let resto = x % y;
    x = y;
    y = resto;
}

console.log("O MDC entre " + a + " e " + b + " é " + x + ".\n");
